superset/connectors/sqla/models.py

```diff
@@ -453,7 +453,7 @@ class SqlaTable(  # pylint: disable=too-many-public-methods,too-many-instance-at
     owner_class = security_manager.user_model
 
     __tablename__ = "tables"
-    __table_args__ = (UniqueConstraint("database_id", "table_name"),)
+    __table_args__ = (UniqueConstraint("database_id", "schema", "table_name"),)
```

Comment:
@john-bodley - do you know if there are any issues with changing __table_args__ here?

Reply:
This will not work per https://github.com/apache/incubator-superset/pull/5449.

Reply:
thanks @john-bodley, reverted this line